### PR TITLE
TIQR-220: eduID theming

### PR DESCRIPTION
**Pull request by bvboldrik**

### CODE REVIEW

File: EduID.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -17,6 +33,22 @@
...
+		0A9F8DBF274FB01D00F24188 /* OFL.txt */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text; path = OFL.txt; sourceTree = "<group>"; };
+		0A9F8DC0274FB01D00F24188 /* SourceSansPro-ExtraLight.ttf */ = {isa = PBXFileReference; lastKnownFileType = file; path = "SourceSansPro-ExtraLight.ttf"; sourceTree = "<group>"; };
+		0A9F8DC1274FB01D00F24188 /* SourceSansPro-BoldItalic.ttf */ = {isa = PBXFileReference; lastKnownFileType = file; path = "SourceSansPro-BoldItalic.ttf"; sourceTree = "<group>"; };
+		0A9F8DC2274FB01D00F24188 /* SourceSansPro-SemiBold.ttf */ = {isa = PBXFileReference; lastKnownFileType = file; path = "SourceSansPro-SemiBold.ttf"; sourceTree = "<group>"; };
+		0A9F8DC3274FB01D00F24188 /* SourceSansPro-ExtraLightItalic.ttf */ = {isa = PBXFileReference; lastKnownFileType = file; path = "SourceSansPro-ExtraLightItalic.ttf"; sourceTree = "<group>"; };
+		0A9F8DC4274FB01D00F24188 /* SourceSansPro-Italic.ttf */ = {isa = PBXFileReference; lastKnownFileType = file; path = "SourceSansPro-Italic.ttf"; sourceTree = "<group>"; };
```
Comment (johankool) [inline EduID.xcodeproj/project.pbxproj:51]: Do you really need all these? Seems excessive. Fonts are said to be relative expensive to load, so would be nice to limit to only what's actually in use.

File: EduID/nl.lproj/InfoPlist.strings [hunk 2]
```diff
@@ -6,5 +6,5 @@
   Copyright © 2016 Egeniq. All rights reserved.
 */
 
-NSCameraUsageDescription = "Tiqr gebruikt de camera om QR-codes te scannen";
-NSFaceIDUsageDescription = "Tiqr gebruikt Face ID om in te loggen zonder een PIN";
+NSCameraUsageDescription = "De eduID app gebruikt de camera om QR-codes te scannen";
+NSFaceIDUsageDescription = "De eduID app gebruikt Face ID om in te loggen zonder een PIN";
```
Comment (johankool) [inline EduID/nl.lproj/InfoPlist.strings:10]: eduID app  ->  eduID-app

File: EduID.xcodeproj/project.pbxproj (hunk 1 above)
Comment (bvboldrik) [reply EduID.xcodeproj/project.pbxproj:51]: the ones that are currently not used are not defined under the fonts key in the info plist, so they are not loaded.

File: EduID/en.lproj/InfoPlist.strings [hunk 3]
```diff
@@ -6,5 +6,5 @@
   Copyright © 2016 Egeniq. All rights reserved.
 */
 
-NSCameraUsageDescription = "The eduID app uses the camera to scan QR-codes";
-NSFaceIDUsageDescription = "The eduID app uses Face ID to login without a PIN";
+NSCameraUsageDescription = "The eduID-app uses the camera to scan QR-codes";
+NSFaceIDUsageDescription = "The eduID-app uses Face ID to login without a PIN";
```
Comment (johankool) [inline EduID/en.lproj/InfoPlist.strings:10]: In English it is a space, in Dutch a hyphen.

**Review by johankool — CHANGES_REQUESTED** [review]

**Review by johankool — APPROVED** [review]

**Pull request merged**